tests/functional/mocha.js

```diff
@@ -65,89 +62,7 @@ registerSuite('mocha tests', {
             });
         }
       })
-      .end()
-
-
-      .then(function () {
-        if (travis) {
-          return sendCoverageToCoveralls(self);
-        } else {
-          return validateCoverageLocally(self);
-        }
-      });
+      .end();
 
   }
 });
-
-/**
- * Sends test coverage data to https://coveralls.io
- * This runs with Travis CI. It pipes "coverageData" gathered from "_$blanket_LCOV" LCOV reporter.
- *
- * @param {Test} context
- * @returns {Deferred}
- */
-function sendCoverageToCoveralls(context) {
```

Comment:
These will come back when https://github.com/mozilla/fxa-content-server/issues/5876 is fixed